superset/daos/base.py

```diff
@@ -133,6 +133,7 @@ def create(
         item: T | None = None,
         attributes: dict[str, Any] | None = None,
         commit: bool = True,
+        add: bool = True,
```

Comment:
I'm concerned about this, because from my understanding this shouldn't be the issue, and I'm worried that we're introducing a pattern that would cause more subtle bugs.

Can we implement this only in `DatabaseDAO.create`, and only for the SSH tunnel case? (ie, don't make it a parameter)

Reply:
You are right that wasn't the issue. This should not be required. I removed it. Thanks!